datadotworld/__init__.py

```diff
@@ -35,7 +35,7 @@
 )
 from datadotworld.datadotworld import DataDotWorld, UriParam  # noqa: F401
 
-__version__ = '1.8.5'
+__version__ = '1.8.6'
```

Comment:
does `1.8.6` feel right here?  I could make arguments lots of ways and haven't thought deeply.

no functionality is changed, but the support matrix of pythons definitely has, so could argue this needs to be `2.0.0` because python<=3.8 is no longer supported.  If so, I'd probably push this as `2.0.0a` so we can maybe fast-follow some more cleanup and fixes, since I think the chances of us finding more incompatibilities not covered by our tests is high, and in any case more modernization would be useful.  Thoughts?

Reply:
>...the support matrix of pythons definitely has, so could argue this needs to be 2.0.0 because python<=3.8 is no longer supported.

I think you've wonderfully outlined the case for making this a `2.0.0` release. 😁 And I'd be ok with making it a `2.0.0a` release, more specifically.